CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - `forbidden_io_function_checker` is now able to check for calls to IO functions written at the top-level of a module, but ouside of the main block.
 
+### New checkers
+
+Custom checkers:
+
+- `top_level_code`: Flag code written at the top level when it is not one of the four acceptable types.
```

Comment:
@ZhiYingSun Oh, I meant you should match the code for the error, which is `forbidden-top-level-code`